File: GerenciaMusic360.Entities/ContractMembers.cs
namespace GerenciaMusic360.Entities
{
    public class ContractMembers
    {
        public int Id { get; set; }
        public int ContractId { get; set; }
        public int PersonId { get; set; }
        public int? CompanyId { get; set; }
        public short ContractRoleId { get; set; }

        public virtual Person Person { get; set; }
        public virtual Company Company { get; set; }
        public virtual ContractRole ContractRole { get; set; }

    }
}

File: GerenciaMusic360/ClientApp/app/shared/models/contractMembers.ts
import { IPerson } from "./person";
import { ILocalCompany } from "./localCompany";
import { IContractRole } from "./contractRole";

export interface IContractMembers {
    id: number;
    contractId: number;
    personId: number;
    companyId: number;
    contractRoleId: number;

    person?: IPerson;
    company?: ILocalCompany;
    contractRole?: IContractRole;
}



